bucketlist/config-sample.py

```diff
@@ -0,0 +1,13 @@
+from os import path
+from datetime import timedelta
+
+ROOT = path.dirname(path.realpath(__file__))
+SQLALCHEMY_DATABASE_URI = 'postgresql://localhost/flask-bucketlist'
+SQLALCHEMY_TRACK_MODIFICATIONS = True
+SECRET_KEY = 'supersecretkey'
```

Comment:
It is not secure to commit your secret key, it should be added in your OS Environment Variables and use the function os.getenv("SECRET_KEY")